wordlists.py

```diff
@@ -0,0 +1,235 @@
+    "burgundy",
+    "camel",
+    "caramel",
+    "canary",
+    "celeste",
+    "cerulean",
+    "champagne",
+    "charcoal",
+    "chartreuse",
+    "chestnut",
+    "chocolate",
+    "citron",
+    "claret",
+    "coal",
+    "cobalt",
+    "coffee",
+    "coral",
+    "corn",
+    "cream",
+    "crimson",
+    "cyan",
+    "denim",
+    "desert",
+    "ebony",
+    "ecru",
+    "emerald",
+    "feldspar",
+    "fuchsia",
+    "gold",
+    "gray",
+    "green",
+    "heather",
+    "indigo",
+    "ivory",
+    "jet",
+    "khaki",
+    "lime",
+    "magenta",
+    "maroon",
+    "mint",
+    "navy",
+    "olive",
+    "orange",
+    "pink",
+    "plum",
+    "purple",
+    "red",
+    "rust",
+    "salmon",
+    "sienna",
+    "silver",
+    "snow",
+    "steel",
+    "tan",
+    "teal",
+    "tomato",
+    "violet",
+    "white",
+    "yellow",
+]
+
+nouns = [
+    "admin",
+    "agreeance",
+    "alignment",
+    "application",
+    "architecture",
+    "availability",
+    "backburner",
+    "bandwidth",
+    "baseline",
+    "benefit",
+    "boondoggle",
+    "brass",
+    "buy-in",
+    "capital",
+    "chain",
+    "channel",
+    "community",
+    "content",
+    "convergence",
+    "coopetition",
+    "cowboy",
+    "creative",
+    "data",
+    "deck",
+    "deliverable",
+    "delta",
+    "dialogue",
+    "disconnect",
+    "dog",
+    "empowerment",
+    "experience",
+    "expertise",
+    "flunky",
+    "functionality",
+    "gatekeeper",
+    "gofer",
+    "goldbricker",
+    "hardball",
+    "idea",
+    "ideation",
+    "imperative",
+    "improvement",
+    "infomediary",
+    "information",
+    "infrastructure",
+    "initiative",
+    "innovation",
+    "interface",
+    "issue",
+    "item",
+    "kicker",
+    "kudos",
+    "language",
+    "leadership",
+    "learning",
+    "linkage",
+    "market",
+    "material",
+    "methodology",
+    "metrics",
+    "mindshare",
+    "model",
+    "network",
+    "niche",
+    "opportunity",
+    "ownership",
+    "paradigm",
+    "partnership",
+    "pivot",
+    "platform",
+    "portal",
+    "potentiality",
+    "practice",
+    "procedure",
+    "process",
+    "product",
+    "pushback",
+    "relationship",
+    "report",
+    "resource",
+    "result",
+    "runway",
+    "scenario",
+    "schema",
+    "scrub",
+    "service",
+    "shrink",
+    "sidebar",
+    "silo",
+    "skill",
+    "skillset",
+    "solution",
+    "source",
+    "space",
+    "strategy",
+    "synergy",
+    "system",
+    "talent",
+    "technology",
+    "upside",
+    "value",
+    "vector",
+    "verbiage",
+    "vision",
+]
+
+webpages = [
+    "access",
+    "bookmark",
+    "browse",
+    "chat",
+    "click",
+    "close",
+    "connect",
+    "download",
+    "drag",
+    "link",
+    "load",
+    "log",
+    "message",
+    "open",
+    "post",
+    "receive",
+    "scroll",
+    "search",
+    "send",
+    "share",
+    "type",
+    "upload",
+    "view",
+    "visit",
+]
```

Comment:
These are not very good, we either replace them now or we create a new issues for the backlog?

Reply:
These are meant to be SEO friendly page names, more like:
aboutus
contactus
blog

Reply:
Reviewed and I have created an issue for the comments above